myapp/db/migrate/20240513042428_create_tasks.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTasks < ActiveRecord::Migration[6.0]
+  def change
+    create_table :tasks do |t|
```

Comment:
it seems like task id won't be created. is it okay?

Reply:
I see "id" column in tasks table. Also, "created_at" and "updated_at". They were created automatically.

Reply:
@kodai-t 
Correct. So id is created automatically and created_at and updated_at are generated by `t.timestamps` 
By the way, could you check the table structure to see if the index for the ID has been created and if it is set to auto-increment?